node-template/runtime/src/lib.rs

```diff
@@ -61,6 +58,9 @@ pub type Hash = primitives::H256;
 /// Digest item type.
 pub type DigestItem = generic::DigestItem<Hash>;
 
+/// The SR25519 pub key of an session that belongs to an Aura authority of the chain.
+pub type AuraId = consensus_aura::sr25519::AuthorityId;
```

Comment:
Hmm I am still not a100% about this stuff but I am almost certain that we don't do this anymore: This type should be defined in Aura primitives crate and imported along the lines of `use aura_primitives::AuthorityId asAuraId`. See how `type AuthorityId` of any of the other modules in `node/runtime` is fulfilled.

Reply:
Actually Aura primitives already defines this on both sr and ed. Based on which one is desired (sr?) it just has to be imported here